### ISSUE TITLE
Is there a "CORS disable option"?

### ISSUE THREAD

**Issue by towith** [kiwibrowser/src.next#601]
Hi, there's option user can choose disable cors in chrome , for pc , add flags `–disable-web-security ` , seems android webview activity can be lanuched by add the same flag in code, there some useful script can works in pc but not due to cors problem.
I try to do it by extension function `declarativeNetRequest`  to rewrite http header but failed, although http header can be rewrite, but target server not support the prelight request  at all.  Wonder if there's workaround to launch kiwi in `cors disabled mode`

Thanks

**Comment by gnadelwartz** [comment]
Did you test if CORS enabling extensions works with kiwi? E.g. https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf

**Comment by towith** [comment]
Yes, I have try this one , and alos `CORS Unblock`  extension, some cors problems indeed can be solved by this solution, seems not work if target sever did not support `OPTIONS` request, such as webdav service return non-200 status for OPTIONS request. I have  try to make an app then called by custom scheme to do this, no cors issue  in cordova app, seems work. It would be better   if there's an option to disable cors check in `kiwi`. Thanks  for your reply , and thanks for your great app :)
